RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -1238,46 +1259,50 @@ string yellowRayCombatString(monster target, boolean inCombat, boolean noForceDr
 	{
 		if((item_amount($item[Yellowcake Bomb]) > 0) && auto_is_valid($item[Yellowcake Bomb]))
 		{
-			return "item " + $item[Yellowcake Bomb];
+			return "item " + $item[Yellowcake Bomb]; // 75 turns
+		}
+		if((item_amount($item[yellow rocket]) > 0) && auto_is_valid($item[yellow rocket]))
+		{
+			return "item " + $item[yellow rocket]; // 75 turns
 		}
 		if(auto_have_skill($skill[Disintegrate]) && (my_mp() >= mp_cost($skill[Disintegrate])))
 		{
-			return "skill " + $skill[Disintegrate];
+			return "skill " + $skill[Disintegrate]; // 100 trurns
 		}
 		if(auto_have_skill($skill[Ball Lightning]) && (my_lightning() >= lightning_cost($skill[Ball Lightning])))
 		{
-			return "skill " + $skill[Ball Lightning];
+			return "skill " + $skill[Ball Lightning]; // 99 turns
 		}
 		if(auto_have_skill($skill[Wrath of Ra]) && (my_mp() >= mp_cost($skill[Wrath of Ra])))
 		{
-			return "skill " + $skill[Wrath of Ra];
+			return "skill " + $skill[Wrath of Ra]; // 100 turns
 		}
 		if((item_amount($item[Mayo Lance]) > 0) && (get_property("mayoLevel").to_int() > 0) && auto_is_valid($item[Mayo Lance]))
 		{
-			return "item " + $item[Mayo Lance];
+			return "item " + $item[Mayo Lance]; // 0 - 145 turns
 		}
 		if((get_property("peteMotorbikeHeadlight") == "Ultrabright Yellow Bulb") && auto_have_skill($skill[Flash Headlight]) && (my_mp() >= mp_cost($skill[Flash Headlight])))
 		{
-			return "skill " + $skill[Flash Headlight];
+			return "skill " + $skill[Flash Headlight]; // 100 turns
 		}
 		foreach it in $items[Golden Light, Pumpkin Bomb, Unbearable Light, Viral Video, micronova]
 		{
 			if((item_amount(it) > 0) && auto_is_valid(it))
 			{
-				return "item " + it;
+				return "item " + it; // ~150 turns
 			}
 		}
 		if(auto_have_skill($skill[Unleash Cowrruption]) && (have_effect($effect[Cowrruption]) >= 30))
 		{
-			return "skill " + $skill[Unleash Cowrruption];
+			return "skill " + $skill[Unleash Cowrruption]; // 149 turns
 		}
 		if((inCombat ? my_familiar() == $familiar[Crimbo Shrub] : auto_have_familiar($familiar[Crimbo Shrub])) && (get_property("shrubGifts") == "yellow"))
 		{
-			return "skill " + $skill[Open a Big Yellow Present];
+			return "skill " + $skill[Open a Big Yellow Present]; // 149 turns
 		}
 		if(inCombat ? have_skill($skill[Unleash the Devil's Kiss]) : possessEquipment($item[unwrapped knock-off retro superhero cape]))
 		{
-			return "skill " + $skill[Unleash the Devil's Kiss];
+			return "skill " + $skill[Unleash the Devil's Kiss]; // 99 turns
 		}
```

Comment:
Shouldn't this be higher priority than any of the 150/149 turn yellow rays?

Reply:
it really should be first. its the best disintegrate. should also change `// 99 turns` to `// 99 turns & 0 MP cost`

Reply:
i took the values right from the wiki didn't verify anything